src/main/java/pl/rpg/menu/Main.java

```diff
@@ -0,0 +1,36 @@
+package pl.rpg.menu;
+
+import pl.rpg.game.Game;
+
+import java.util.Scanner;
+
+public class Main {
+
+  public static void main(String[] args) {
```

Comment:
Main method should be in main class (pl.rpg.game). We should not commit additional main methods to git. Writing code in main method is HUGE mistake. We live in object world so You should probably create class for this code ;)